### PR TITLE
Make key-features description more specific

### PR DESCRIPTION
**Pull request by loziniak**
According to: https://github.com/exercism/red/pull/17/files#r599312829

### CODE REVIEW

File: building/tracks/config-json.md [hunk 1]
```diff
@@ -233,7 +233,7 @@ Each concept is an entry in the top-level `concepts` array. The following fields
 
 ## Key features
 
-The language's key features succinctly describe what the most important features of the language are.
+The language's key features succinctly describe what the most important features of the language are. They are meant to "sell" a language in a way, getting students interested in the language. Easily understandable titles are most beneficial.
```
Comment (iHiD) [inline building/tracks/config-json.md]:
```suggestion
The language's key features succinctly describe what the most important features of the language are. 
They are intended to upsell the more interesting features of a language to potential students. 
Title's should aim to be quickly understandable with as little technical jargin as possible, bearing a mind a student may not understand the language features until they have learnt the language.
```

**Comment by iHiD** [comment]
Thanks :) I've updated it slightly. I'll get a second opinion from @ErikSchierboom before merging too :)

File: building/tracks/config-json.md [hunk 2]
```diff
@@ -233,7 +233,9 @@ Each concept is an entry in the top-level `concepts` array. The following fields
 
 ## Key features
 
-The language's key features succinctly describe what the most important features of the language are.
+The language's key features succinctly describe what the most important features of the language are. 
+They are intended to upsell the more interesting features of a language to potential students. 
+Title's should aim to be quickly understandable with as little technical jargin as possible, bearing a mind a student may not understand the language features until they have learnt the language.
```
Comment (ErikSchierboom) [inline building/tracks/config-json.md]: Maybe simplify this a bit to something like:

```suggestion
Titles should strive to use as little technical jargon as possible, bearing in mind that students might not be familiar with what language-specific jargon means before learning that language.
```

**Review by ErikSchierboom — COMMENTED** [review]
Great! One small nit

**Pull request merged**